components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -49,6 +49,13 @@
  */
 @property(nonatomic, assign) CGFloat dialogCornerRadius;
 
+/**
+ Customize the color of the background scrim.
+
+ Defaults to a semi-transparent White.
```

Comment:
This looks like it might be a typo - I think this should be "semi-transparent black" based on this code:

https://github.com/material-components/material-components-ios/blob/7672bcf7a61d48af4da9947efdbbe2d34d241b09/components/Dialogs/src/MDCDialogPresentationController.m#L68-L70

Reply:
It's counter intuitive to thing that colorWithWhite:0 is actually black....
Good catch :)